index.d.ts

```diff
@@ -28,13 +28,15 @@ export type Options = {
 
  	@default 'GH-'
 
+	@example
  	```
```

Comment:
Are .d.ts files excluded from XO? There are a couple of `<space><tab>` instances here

Reply:
Strange. Will look into it.